src/compiler/BUILD

```diff
@@ -69,6 +69,7 @@ grpc_cc_library(
         "schema_interface.h",
     ],
     external_deps = [
+        "absl/strings",
```

Comment:
Can you exclude this from the dependency and implement replace_all without absl? This is because code_generator's build is a bit tighter so it cannot depend on abseil sometimes. We might be able to address this but it'd be quicker to change it not to rely on it.